builtin/rebase--helper.c

```diff
@@ -37,7 +55,7 @@ int cmd_rebase__helper(int argc, const char **argv, const char *prefix)
 	int abbreviate_commands = 0, rebase_cousins = -1, ret;
 	const char *head_hash = NULL, *onto = NULL, *restrict_revisions = NULL,
 		*squash_onto = NULL, *upstream = NULL;
-	char *revisions = NULL;
+	char *revisions = NULL, *shortrevisions = NULL;
```

Comment:
It seems to me that the declarations of revisions and shortrevisions could be moved below inside the if clauses where these variables are used.